src/renderer/components/+storage-classes/storage-class-details.tsx

```diff
@@ -10,14 +10,22 @@ import { KubeObjectDetailsProps } from "../kube-object";
 import { StorageClass } from "../../api/endpoints";
 import { KubeObjectMeta } from "../kube-object/kube-object-meta";
 import { kubeObjectDetailRegistry } from "../../api/kube-object-detail-registry";
+import { storageClassStore } from "./storage-class.store";
+import { VolumeDetailsList } from "../+storage-volumes/volume-details-list";
+import { volumesStore } from "../+storage-volumes/volumes.store";
 
 interface Props extends KubeObjectDetailsProps<StorageClass> {
 }
 
 @observer
 export class StorageClassDetails extends React.Component<Props> {
+  async componentDidMount() {
+    volumesStore.reloadAll();
```

Comment:
Shouldn't this use the new `subscribe` mechanism @ixrock ?

Reply:
Probably no since it's details panel.. @jakolehm wdyt?